google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -97,10 +97,13 @@ def get_group_members(self, group_key):
         request = members_stub.list(groupKey=group_key,
                                     maxResults=FLAGS.max_results_admin_api)
 
-        results = self._build_paged_result(
+        paged_results = self._build_paged_result(
             request, members_stub, self.rate_limiter)
 
-        return [item.get('members', []) for item in results]
+        group_members = []
```

Comment:
Okay, I added a TODO and issue #263 for this, as I think it should go in the _base_client.py, and will be a bigger change than can be done quickly here.  Also, we should write tests for this.  :)